content/v2.0/query-data/optimize-queries.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Optimize Flux queries
+description: >
+  Optimize your Flux queries by changing function order, understanding how functions work,
+  and being conscious of the nature of the data queried.
+weight: 104
+menu:
+  v2_0:
+    name: Optimize queries
+    parent: Query data
+v2.0/tags: [query]
+---
+
+Optimize your Flux queries with a few simple principles.
+Query optimizations center around reducing the memory and compute (CPU) requirements
+of a query by changing function order, understanding how functions work, and being
+conscious of the nature of the data queried.
+
+## Start queries with pushdown functions
```

Comment:
Scott, maybe replace the bullets above with headings:
- Start queries w...
- Increase window duration/intervals?
- Limit? "heavy" functions (maybe be explicit here: specified transformation functions?)